common/src/main/scala/explore/components/ui/ExploreStyles.scala

```diff
@@ -371,6 +371,9 @@ object ExploreStyles {
   val TargetSummarySubRowCell: Css = Css("explore-target-summary-subrow-cell")
   val ConstraintsSummaryEdit: Css  = Css("explore-constraints-summary-edit")
 
+  object Dialog: // SUI has Mini, Tiny, Small, Large and Fullscreen.
+    val Small: Css = Css("pl-dialog-small")
```

Comment:
Should these go to `PrimeStyle` eventually?

Reply:
I don't know. The CSS depends on the breakpoints we set in explore. It doesn't seem like the `Css` should go there if it isn't backed by actual styles.

Reply:
It's possible we could move it all to Lucuma UI to be used across lucuma projects.